### ISSUE TITLE
milestone 4

### ISSUE THREAD

**Issue by sukh2929** [UBC-MDS/group29#77]

**Comment by sukh2929** [comment]
1.create dependency diagram - should be in the project directory 
2.create docker file - should be in the project directory
3.create docker image
4.update readme to add the link of dependency diagram
5.update readme to use your project with and without Docker

**Comment by rachelywong** [comment]
According to TA and peer feedback this is what we need to change for script5:

1. - explain why we used balanced log regression (explain when we talk about the model scores)
2. - remove scatterplot matrix?
3. - explain sampling (why we chose to use 1000)
4. - be more explicit about which features we ended up keeping and dropping
5. - fix conclusion about model scores and overall conclusion (especially part about logistic regression balanced performing average)
6. - talk more about figure 1 in results and discussion
7. - add why fit and score times, accuracy, and f1 scores were chosen as evaluation metrics
8. - grammar check

**Comment by jraza19** [comment]
edits to the read me after making the dockerfile include:

- add random as a dependency 
- change the project proposal last line
- add the new docker code to the readme file 
- create new dependency diagram

**Comment by rachelywong** [comment]
add sentence about certain sample we pick with random.seed and how data could be missing

**Comment by jraza19** [comment]
edit to eda2.py to address TA feedback: 

# Split the data into training (0.8) and testing (0.2)
diabetes_with_race = pd.read_csv(datafile[1])
train_df, test_df = train_test_split(diabetes_with_race, test_size=0.2, random_state=123)
diabetes_subset = train_df
